File: day 31/Homework/goa1.py
def separate_evens_odds(numbers):
    result = []
    for number in numbers:
        if number % 2 == 0:
            result.append(number // 2)
        else:
            result.append(number * 2)
    return result


print(separate_evens_odds([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]))

File: day 112/Homework/goa1.js
class MathHelper {
    static PI = Math.PI;
    static E = Math.E;
    static DEFAULT_PRECISION = 3;
  
    static factorial(n) {
      if (n < 0) {
        throw new Error("Negative numbers are not allowed");
      }
      let result = 1;
      for (let i = 1; i <= n; i++) {
        result *= i;
      }
      return result;
    }
  
    static endsWith5Or0(n) {
      return n % 10 === 0 || n % 10 === 5;
    }
  
    static roundToNearest5Or0(n) {
      return Math.round(n / 5) * 5;
    }
  
    static roundToDecimalPlaces(n, precision = MathHelper.DEFAULT_PRECISION) {
      return n.toFixed(precision);
    }
  }
  